install-deps.sh

```diff
@@ -493,6 +500,9 @@ EOF
                 fi
                 ;;
         esac
+        if [ "$INSTALL_EXTRA_PACKAGES" ]; then
+            $SUDO dnf install -y $INSTALL_EXTRA_PACKAGES
```

Comment:
my concern is, what if the package names are different in debian derivatives and the rpm based distros?

Reply:
In this case, the script/program setting INSTALL_EXTRA_PACKAGES will be responsible for passing the correct package names for the distro. This was the case before as well. The advantage of INSTALL_EXTRA_PACKAGES is that install-deps.sh script can order the install & manage the system in one central place.

For what it's worth, my WIP branch is working on both ubutnu and centos 8 stream at this time.